### PR TITLE
Use TeeEnclaveConfiguration builder in tests

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionMakerServiceTests.java [hunk 1]
```diff
@@ -46,13 +45,12 @@ void beforeEach() {
     // region getSessionYml
     @Test
     void shouldGetSessionJson() throws Exception {
-        TeeEnclaveConfiguration enclaveConfig = mock(TeeEnclaveConfiguration.class);
+        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
+                .fingerprint(APP_FINGERPRINT).entrypoint("/apploader.sh").build();
```
Comment (mcornaton) [inline src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionMakerServiceTests.java:49]: I find it clearer on multiple lines.
```suggestion
        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
                .fingerprint(APP_FINGERPRINT)
                .entrypoint("/apploader.sh")
                .build();
```

File: src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java [hunk 2]
```diff
@@ -72,13 +71,12 @@ void beforeEach() {
     // region getSessionYml
     @Test
     void shouldGetSessionYml() throws Exception {
-        TeeEnclaveConfiguration enclaveConfig = mock(TeeEnclaveConfiguration.class);
+        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
+                .fingerprint(APP_FINGERPRINT).entrypoint(APP_ENTRYPOINT).build();
```
Comment (mcornaton) [inline src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java:75]: I find it clearer on multiple lines.
```suggestion
        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
                .fingerprint(APP_FINGERPRINT)
                .entrypoint(APP_ENTRYPOINT)
                .build();
```

**Review by mcornaton — APPROVED** [review]

File: src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionMakerServiceTests.java (hunk 1 above)
Comment (jbern0rd) [reply src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionMakerServiceTests.java]: done in c1d5287

File: src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java (hunk 2 above)
Comment (jbern0rd) [reply src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java]: done in c1d5287

**Review by jeremyjams — APPROVED** [review]

**Pull request merged**